spec/support/helpers.rb

```diff
@@ -27,7 +27,7 @@ def stub_omniauth(disable_completely: true, user: nil)
     end
   end
 
-  def stub_session_create(user: double(id: 1))
+  def stub_session_create(user: double(id: 1, 'opted_in?': false))
```

Comment:
Everyone's opted in now, I think this should default to `true`

Reply:
Agreed. I'm making the bigger change here - https://github.com/DFE-Digital/manage-courses-frontend/pull/244